rpcs3/Emu/Cell/lv2/sys_dbg.cpp

```diff
@@ -1,11 +1,127 @@
 #include "stdafx.h"
-#include "Emu/Memory/Memory.h"
-#include "Emu/System.h"
-#include "Emu/IdManager.h"
 
 #include "Emu/Cell/ErrorCodes.h"
+#include "Emu/Memory/Memory.h"
 #include "sys_dbg.h"
 
+logs::channel sys_dbg("sys_dbg");
 
+error_code sys_dbg_read_process_memory(s32 pid, u32 address, u32 size, vm::ptr<void> data)
+{
+	sys_dbg.warning("sys_dbg_read_process_memory(pid=0x%x, address=0x%llx, size=0x%x, data=*0x%x)", pid, address, size, data);
 
-logs::channel sys_dbg("sys_dbg");
+	// Todo(TGEnigma): Process lookup (only 1 process exists right now)
+	if (pid != 1)
+	{
+		return 0x80010409;
```

Comment:
Return named error codes instead of an unnamed constant, in this case use CELL_LV2DBG_ERROR_DEINVALIDARGUMENTS

Reply:
Fixed